src/settings.cpp

```diff
@@ -316,6 +316,34 @@ void Settings::setExecutablesBlacklist(const QString& s)
   set(m_Settings, "Settings", "executable_blacklist", s);
 }
 
+QStringList Settings::skipFileSuffixes() const
```

Comment:
Do not convert the `QStringList` to a `QString` for storage, directly store the `QStringList`, Qt will take care of escaping the right characters when needed (applies to the 4 methods).